### DISCUSSION TITLE
Delta

### DISCUSSION THREAD

**Discussion by MAHcodes** [catppuccin/catppuccin#1829] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
Add any other context or screenshots about the port request here.
[delta](https://github.com/dandavison/delta)
[theming examples](https://github.com/dandavison/delta/blob/master/themes.gitconfig)

**Comment by dvic** [comment]
@MAHcodes did you happen to start working on this?

**Comment by zyhou** [comment]
Delta can use bat theme, https://dandavison.github.io/delta/supported-languages-and-themes.html so we can use https://github.com/catppuccin/bat

```properties
[delta]
	syntax-theme = Catppuccin-mocha # Use bat theme
```

**Reply by dvic** [reply]
Nice, didn't know about this one!

**Reply by ming-hao-xu** [reply]
But this only set syntax highlighting to `Catppuccin`, I wish there is a 'theme' too like [themes.gitconfig](https://github.com/dandavison/delta/blob/main/themes.gitconfig)

**Reply by zyhou** [reply]
Ah true, agree could be nice to have a delta theme

**Comment by MAHcodes** [comment]
I found [mellow-barbet theme](https://github.com/dandavison/delta/blob/fdfcc8fce30754a4f05eeb167a15d519888fc909/themes.gitconfig#L442-L475), works really well with my terminal colors, I'm using [alacritty](https://github.com/alacritty/alacritty) + [catppuccin theme](https://github.com/catppuccin/alacritty)
![image](https://github.com/catppuccin/catppuccin/assets/69465962/dfa968f4-ed7e-4a19-979f-b263334c5523)

my [dotfiles](https://github.com/MAHcodes/dotfiles)

**Comment by Anomalocaridid** [comment]
I just made a port for delta and submitted it for review: #2246

**Comment by sgoudham** [comment]
Closing as any further discussion should continue in #2246
